test/test_logger.py

```diff
@@ -281,12 +283,16 @@ def test_apprise_log_file_captures(tmpdir):
 
         assert len(logs) == 5
 
-        # Remove our file before we exit the with clause
-        # this causes our delete() call to throw gracefully inside
-        os.unlink(str(log_file))
+        # Concurrent file access is not possible on Windows.
+        # PermissionError: [WinError 32] The process cannot access the file
+        # because it is being used by another process.
+        if sys.platform != "win32":
+            # Remove our file before we exit the with clause
+            # this causes our delete() call to throw gracefully inside
+            os.unlink(str(log_file))
 
-        # Verify file is gone
-        assert not os.path.isfile(str(log_file))
+            # Verify file is gone
+            assert not os.path.isfile(str(log_file))
```

Comment:
I don't know what was intended to be tested here, but like it's done here will croak on Windows. The reason is because concurrent file access is not possible on Windows.
```
PermissionError: [WinError 32] The process cannot access the file
because it is being used by another process.
```
43dd1be70 fixes it.